### DISCUSSION TITLE
Building Junicode

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#320] [Q&A]
When I run `build_font` it tries to build Junicode-Ansund* fonts and then fails because of the lack of the appropriate *.xgf files. What am I doing wrong?

**Comment by psb1558** [comment]
I think you had the bad luck to download a specialized version that I uploaded by accident some time ago. The version now in the repo is correct.

It sounds like you are building the static version. The error about .xgf files has to do with TrueType hinting. I suggest that you build the .otf version (use the -o option) instead of .ttf, as the autohinter for .otf works better for Junicode than the one for .ttf. Or, to skip the hassle of hinting, use the -n option.

**Comment by jsbien** [comment]
Thank you very much!
